generator/internal/mock_connection_generator.cc

```diff
@@ -103,10 +103,15 @@ class $mock_connection_class_name$ : public $product_namespace$::$connection_cla
                  Not(IsPaginated))),
          MethodPattern(
              {
+                 {// clang-format off
+    "\n  // Due to additional overloads for this method\n"
+    "  // EXPECT_CALL(*mock, $method_name$) is now ambiguous. Use\n"
+    "  // EXPECT_CALL(*mock, $method_name$(_)) instead.\n"},
```

Comment:
We should use doxygen style comments `///` and put the code in code quotes.

And optionally, spell out `::testing::_`.